app/assets/v2/js/pages/new_bounty.js

```diff
@@ -295,7 +384,8 @@ $(document).ready(function() {
         repo_type: data.repo_type,
         featuring_date: data.featuredBounty && ((new Date().getTime() / 1000) | 0) || 0,
         reservedFor: reservedFor ? reservedFor.text : '',
-        tokenName
+        tokenName,
+        invite: inviteContributors
```

Comment:
I'm sending an array of usernames ['octavioamu', 'saptak']
I'm suppose you are going to get this data to make the email sending.